spec/controllers/players_controller_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
+
+RSpec.describe PlayersController do
```

Comment:
Adding a single controller spec that covers most of the files, like the Model, Serializer, and Filter Service

Reply:
<img width="1437" alt="Screen Shot 2022-08-23 at 3 06 22" src="https://user-images.githubusercontent.com/55289038/186106565-7779950c-5335-4bbf-b59d-f37399ba718c.png">